drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -117,16 +111,13 @@ static uint32_t get_sysclk_frequency(void)
 	__ASSERT(0, "No SYSCLK Source configured");
 	return 0;
 #endif
-
 }
 
 /** @brief Verifies clock is part of active clock configuration */
 int enabled_clock(uint32_t src_clk)
 {
-	if ((src_clk == STM32_SRC_SYSCLK) ||
-	    (src_clk == STM32_SRC_HCLK) ||
-	    (src_clk == STM32_SRC_PCLK1) ||
-	    (src_clk == STM32_SRC_PCLK2) ||
+	if ((src_clk == STM32_SRC_SYSCLK) || (src_clk == STM32_SRC_HCLK) ||
+	    (src_clk == STM32_SRC_PCLK1) || (src_clk == STM32_SRC_PCLK2) ||
 	    (src_clk == STM32_SRC_PCLK3) ||
 	    ((src_clk == STM32_SRC_HSE) && IS_ENABLED(STM32_HSE_ENABLED)) ||
 	    ((src_clk == STM32_SRC_HSI16) && IS_ENABLED(STM32_HSI_ENABLED)) ||
```

Comment:
This change makes code harder to read and maintain.

Reply:
abandoned